warehouse/models/mart/gtfs_quality/_mart_gtfs_quality.yml

```diff
@@ -26,6 +26,9 @@ models:
       - name: code
         tests:
           - not_null
+      - name: description
```

Comment:
kind of a nitpick but in the spirit of handoff, can we add a description here? I'm trying to be more aggressive about documenting everything at this point.